pkg/cluster/reconcile.go

```diff
@@ -100,26 +96,8 @@ func (c *Cluster) reconcileMembers(running etcdutil.MemberSet) error {
 		return c.removeDeadMember(c.members.Diff(L).PickOne())
 	}
 
-	// quorum is lost and cluster shouldn't be automatically recovered, fail
-	if !c.config.RecoverQuorumLoss {
-		return ErrLostQuorum
-	}
-
-	// quorum is lost, autorecover //
-
-	// Remove any still live Pods
-	for l := range L {
-		if err := c.removePod(l); err != nil {
-			return err
-		}
-	}
-
-	// Reinitialize the cluster again
-	if err := c.create(); err != nil {
-		return err
-	}
```

Comment:
This would get cluster into wrong Creating phase; transition to Running wasn't possible, just to Failed and even that only on next etcd-operator Pod start.